macOS/UnitTests/DataImport/FirefoxDataImporterTests.swift

```diff
@@ -82,7 +82,7 @@ class FirefoxDataImporterTests: XCTestCase {
 
         _ = await importer.importData(types: [.bookmarks])
 
-        XCTAssertEqual(bookmarksToImport?.numberOfBookmarks, 10)
```

Comment:
I added another site (with an amazon.com URL) to the history database in the Firefox test data, for a new test in `FirefoxHistoryReaderTests`. This is imported as another favorite bookmark, increasing the total number of bookmarks.